chapter05/ard/ex1.pl

```diff
@@ -0,0 +1,19 @@
+# Declaring a hash using '%' but using braces which denote an anonymous hash - oops!
+my %passenger_1 = {
+  name => 'Ginger',
+  age => 22,
+  occupation => 'Movie Star',
+  real_age => 35,  # Ginger is a liar.
+  hat => undef,
+};
+
+# Repeat offence!
+my %passenger_2 = {
+  name => 'Mary-Ann',
+  age => 19,
+  hat => 'bonnet',  # Nobody over the age of 3 should be allowed to wear a bonnet.
```

Comment:
Isn't there a get out clause to that rule, specifically for people named _Mary-Ann_? It just _sounds_ like a bonnet name.